application/src/main/java/de/wuespace/telestion/project/daedalus2/MessageTransformer.java

```diff
@@ -0,0 +1,48 @@
+package de.wuespace.telestion.project.daedalus2;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import de.wuespace.telestion.api.config.Config;
+import de.wuespace.telestion.api.message.JsonMessage;
+import de.wuespace.telestion.extension.mavlink.message.MavlinkMessage;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Promise;
+
+public class MessageTransformer extends AbstractVerticle {
+	private Configuration configuration;
```

Comment:
Correct implementation of configuration. Well done! But you could rename this variable to `config` to be consistent with other classes.